src/controllers/OrganisationsController.js

```diff
@@ -128,44 +127,31 @@ const organisationsController = {
   },
 
   async getDatasetTaskList (req, res, next) {
-    const issues = [
-      {
-        num_issues: 2,
-        issue_type: 'future entry date',
-        resource: 'resource1',
-        status: 'Error'
-      },
-      {
-        num_issues: 3,
-        issue_type: 'invalid coordinates',
-        resource: 'resource2',
-        status: 'Issue'
-      },
-      {
-        num_issues: 1,
-        issue_type: 'invalid decimal',
-        resource: 'resource3',
-        status: 'Warning'
-      }
-    ]
+    try {
+      const lpa = req.params.lpa
+      const datasetId = req.params.dataset
 
-    const taskList = getTaskList(issues)
+      const organisationResult = await datasette.runQuery(`SELECT name FROM organisation WHERE organisation = '${lpa}'`)
+      const organisation = organisationResult.formattedData[0]
 
-    const organisation = {
-      name: "George's fake organisation"
-    }
+      const datasetResult = await datasette.runQuery(`SELECT name FROM dataset WHERE dataset = '${datasetId}'`)
+      const dataset = datasetResult.formattedData[0]
 
-    const dataset = {
-      name: 'Article 4 direction area'
-    }
+      const issues = await performanceDbApi.getLpaDatasetIssues(lpa, datasetId)
 
-    const params = {
-      taskList,
-      organisation,
-      dataset
-    }
+      const taskList = performanceDbApi.getTaskList(issues)
+
+      const params = {
+        taskList,
+        organisation,
+        dataset
+      }
 
-    res.render('organisations/datasetTaskList.html', params)
+      res.render('organisations/datasetTaskList.html', params)
+    } catch (e) {
+      logger.error(e)
```

Comment:
I assume this is guarding mostly for the datasette errors (which `performanceDbApi.*` also calls. `runQuery` already does `logger.warn(error)`, so it would be useful to give some more context here. 

```
import {types} from '../utils/logging.js'

logger.warn(`getDAtasetTaskList() failed for lpa='${lpa}', datasetId='${datasetId}'`, {type: types.App})

```

I've set the 'type' to 'App', because that try block covers more than just remote service interactions.